integration/commands_administration_test.go

```diff
@@ -193,3 +198,132 @@ func TestCommandsAdministrationGetParameter(t *testing.T) {
 	assert.Contains(t, keys, "quiet")
 	assert.Equal(t, false, m["quiet"])
 }
+
+func TestStatisticsCommands(t *testing.T) {
+	t.Skip("TODO: response deltas investigation needed")
+	ctx, collection := setup(t, shareddata.Scalars, shareddata.Composites)
+
+	for name, tc := range map[string]struct {
+		command  any
+		response map[string]any
```

Comment:
Unless it is very hard, we should compare `bson.D`s to also check fields order

Reply:
I used `map[string]any` to skip fields like `gitVersion` or `debug` field which requires me to use `util/version` package.
If it's ok to use it then I can change `response` field to `bson.D`.

Reply:
I've changed to `bson.D` for now.